reference/performance.mdx

```diff
@@ -5,36 +5,288 @@ sidebarTitle: Performance
+
+### Why it matters
+
+- **Poor performance on time-based queries**: Blocks must be scanned despite time filters, because many blocks overlap the query time window.
+- **Inefficient block filtering**: Queries that filter on time must scan blocks that contain data from many time segments.
+- **Large data merges**: Compaction processes that rely on time ordering become less efficient.
+
+### How to fix it
+
+- **Minimize backfill:** Try to ingest events close to their actual creation time whenever possible. Ingest events close to the time they occur.
+- **Backfill in dedicated batches:** If you must backfill older data, do it in dedicated batches that do not mix with live data.
+- **Use discrete backfill intervals:** When backfilling data, ingest one segment at a time (for example, day-by-day).
+- **Avoid wide time ranges in a single batch:** If you are sending data for a 24-hour period, avoid mixing in data that is weeks or months older.
+- **Be aware of ingestion concurrency:** Avoid mixing brand-new events with extremely old events in the same ingest request.
+
+<Note>
+Future improvements: Axiom’s roadmap includes an initiative which aims to mitigate the impact of poorly clustered time data by performing incremental time-based compaction. Until then, avoid mixing large historical ranges with live ingest whenever possible.
+</Note>
+
+## Large number of fields in a dataset
+
+### Problem
+
+Slow query performance in datasets with very high dimensionality (with more than several thousand fields).
+
+### Why it matters
 
 Axiom stores event data in a tuned format. As a result:
 - The number of distinct values (cardinality) in your data has little impact on performance.
-- The number of fields in a dataset (dimensionality) does impact performance.
+- The number of fields in a dataset (dimensionality) impacts performance.
 - The volume of data collected impacts performance.
 
+### How to fix it
+
 Scoping the number of fields in a dataset below a few thousand can help you achieve the best performance in Axiom.
 
-## Optimize queries
+## Failing to use the `_time` field for event timestamps
+
+### Problem
+
+Axiom’s core optimizations rely on `_time` for indexing and time-based queries. If you store event timestamps in a different field (for example, `timestamp` or `created_at`) and use that field in time filters, Axiom’s time-based optimizations will not be applied.
+
+### Why it matters
+
+- **No time-based indexing**: Every block must be scanned, because your custom timestamp field is invisible to the time index.
+- **Incorrect event ordering**: You may lose correct chronological context or get unexpected query results.
```

Comment:
This is inaccurate. We do maintain correct sorting order on any field we explicitly sort on, it just won't be efficient.